habit/src/reducers/root-reducer.js

```diff
@@ -0,0 +1,5 @@
+import {combineReducers} from 'redux';
```

Comment:
We can shift this code into the index.js file of root-reducer

Reply:
Every folder has an index.js file, which is the first file called when importing something from there.
So, using the index.js file will reduce our imports from `import rootReducer from '../reducers/root-reducer';` to `import rootReducer from '../reducers/';`